kitchen/src/web/storage/mod.rs

```diff
@@ -256,6 +256,7 @@ pub struct SqliteStore {
 
 impl SqliteStore {
     pub async fn new<P: AsRef<Path>>(path: P) -> sqlx::Result<Self> {
+        std::fs::create_dir_all(&path)?;
```

Comment:
I'm not entirely sure it *should* create the directory. In a sense if you point it at the wrong directory and it *just* works that's an error. My test run script assumes that directory exists but it could just as easily create it.

Perhaps the correct fix is to make this a required flag rather than having a default?